src/DynamicMvvm.Tests/Command/DynamicCommandBuilderFactoryTests.cs

```diff
@@ -155,6 +156,16 @@ public async Task It_applies_strategies_in_order2()
 			testString.Should().Be("123");
 		}
 
+		[Fact]
+		public void It_passes_ViewModel_owner_correctly()
```

Comment:
Don't we wanna also test (adding a new test) if it does not pass, that the good exception is thrown ?

Reply:
null is a correct value. I added a test that validates that :)